internal/model/toc.go

```diff
@@ -68,6 +68,26 @@ type IndexVersion struct {
 	ExternalID  string            `json:"externalID"`
 }
 
+func (idx *Index) IsEmpty() bool {
+	return len(idx.Data) == 0
+}
+
+func (idx *Index) Sort() {
+	if idx.IsEmpty() {
+		return
+	}
+	// sort versions of each entry ascending
+	for _, entry := range idx.Data {
+		slices.SortFunc(entry.Versions, func(a IndexVersion, b IndexVersion) int {
+			return strings.Compare(a.TMID, b.TMID)
```

Comment:
I'd suggest sorting the versions in the proper descending order rather than ascending lexicographically. See https://github.com/wot-oss/tmc/blob/cc71c5b3bb7aaf39e8272256028f62019a58c278/internal/commands/fetch.go#L244

Reply:
To be consistently, versions are now ordered descending also in index.